ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+        <field name="office_365.RecordType" type="osregex">^89$</field>
+        <description>Office 365: Events related to admin actions in Automated investigation and response (AIR).</description>
+        <options>no_full_log</options>
+        <group>AirAdminActionInvestigation</group>
+    </rule>
+
+    <rule id="91615" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^90$</field>
+        <description>Office 365: Threat intelligence events in Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>MSTIC</group>
+    </rule>
+
+    <rule id="91616" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^91$</field>
+        <description>Office 365: Events related to physical badging signals that support the Insider risk management solution.</description>
+        <options>no_full_log</options>
+        <group>PhysicalBadgingSignal</group>
+    </rule>
+
+    <rule id="91617" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^93$</field>
+        <description>Office 365: Azure Information Protection (AIP) scanner events.</description>
+        <options>no_full_log</options>
+        <group>AipDiscover</group>
+    </rule>
+
+    <rule id="91618" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^94$</field>
+        <description>Office 365: AIP sensitivity label events.</description>
+        <options>no_full_log</options>
+        <group>AipSensitivityLabelAction</group>
+    </rule>
+
+    <rule id="91619" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^95$</field>
+        <description>Office 365: AIP protection events.</description>
+        <options>no_full_log</options>
+        <group>AipProtectionAction</group>
+    </rule>
+
+    <rule id="91620" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^96$</field>
+        <description>Office 365: AIP file deletion events.</description>
+        <options>no_full_log</options>
+        <group>AipFileDeleted</group>
+    </rule>
+
+    <rule id="91621" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^97$</field>
+        <description>Office 365: AIP heartbeat events.</description>
+        <options>no_full_log</options>
+        <group>AipHeartBeat</group>
+    </rule>
+
+    <rule id="91622" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^98$</field>
+        <description>Office 365: Events corresponding to alerts triggered by Microsoft Cloud App Security.</description>
+        <options>no_full_log</options>
+        <group>MCASAlerts</group>
+    </rule>
+
+    <rule id="91623" level="3">
```

Comment:
I think this level should be higher since these alerts come from sensitive data.

Reply:
Done